src/lib/validationRules/index.ts

```diff
@@ -0,0 +1,162 @@
+//
```

Comment:
Best to lift this file up one directory. This file should probably just export two arrays:

```
export const localRules = [ ... ]
export const remoteRule = [ ... ]
```

And then in the `lib/` directory you'd have a `validateRemoteDocument.ts` and `validateLocalDocument.ts` file.